packages/interactivity/src/init.ts

```diff
@@ -11,11 +11,11 @@ import { directivePrefix } from './constants';
 
 // Keep the same root fragment for each interactive region node.
 const regionRootFragments = new WeakMap();
-export const getRegionRootFragment = ( region ) => {
+export const getRegionRootFragment = ( region: Element ): ContainerNode => {
 	if ( ! regionRootFragments.has( region ) ) {
 		regionRootFragments.set(
 			region,
-			createRootFragment( region.parentElement, region )
+			createRootFragment( region.parentElement!, region )
```

Comment:
I'm not sure if this is a good practice. I guess it would be better to throw an error if `region` has no `parentElement`, instead of waiting for the runtime to fail later on. 🤔

Reply:
Yes, I think that would be much better. Throw an error. I _think_ this would that the region was `html` which we don't want.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/61723/commits/1a8f4dea463de41438fac7fef3fdc31cb23e339d.